AAI/AAIConnectProfile.md

```diff
@@ -701,6 +710,8 @@ Issuer.
    -   `<ga4gh-spec-claims>`: OPTIONAL. One or more GA4GH Claims MAY be
        provided. See [Authorization/Claims](#authorizationclaims) for an
        example.
+       
+   - MAY contain `aud` to list approved brokers
```

Comment:
micro-nit: please add 2 spaces after the dash ("-") to align with the other entries above and add period punctuation at the end for consistency.